_rules/heading-name-does-not-only-contain-spaces-ffd0e9.md

```diff
@@ -0,0 +1,190 @@
+---
+id: ffd0e9
+name: Heading does not only consist of seperators or breaks
+rule_type: atomic
+description: | 
+  This rule checks that heading elements do not have Unicode separator characters or `br` elements as their only content.
+
+success_criterion: 
+- 1.3.1 # Info and Relationships (A)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+- Kasper Isager
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to any HTML element with the [semantic role](#semantic-role) of `heading` that is [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+### Expectation
+
+The target element either:
+- contains [text nodes](https://www.w3.org/TR/dom/#text) that do not only consist of [Unicode separator characters](https://www.unicode.org/versions/Unicode11.0.0/ch04.pdf#G134153), or 
+- does not contain any HTML `<br>` elements, or 
```

Comment:
This should not be enough to pass the expectation on it's own.

This would (for example) pass "failed example 2"

Reply:
> 
> 
> This should not be enough to pass the expectation on it's own.
> 
> This would pass "failed example 2" 3, 4, 5, 6, 7, and 9.